RedHatInsights/insights-inventory-frontend
# ESSNTL(4056): Adds checks against edge cases for last seen filter

To test new updates check for the following:

- When selecting last seen filter, if you refresh, filter persist
- Make sure you can copy/past the link with the filter present (this is true for all except custom, custom should populate the range picker)
- Make sure Start Date cant be larger than End Date
- Make sure End date cant be smaller than start Date
- End date and start date both cannot go past Todays date

The only current way to select the same date in both fields, is to

- Select a date on the EndDate 
- Invalidate that date (write something like 2023-10-00)
- Then input the date originally typed in on the End that (the end date just say 'invalid date' as you do this) on the Start Date

The filter will load both args into the request, but it should Not break. 
Other than this edge case there shouldnt be a way to select the same day on both (if there is some other hacky way, it should not break)
Filter should no break ever. I believe this should catch all edge cases
